src/k8s/bpd_configmaps.tf

```diff
@@ -136,11 +136,11 @@ resource "kubernetes_config_map" "bpdmsnotificationmanager" {
     NOTIFICATION_SERVICE_END_PERIOD_SCHEDULE                          = "-"
     NOTIFICATION_SERVICE_NOTIFY_PAYMENT_WINNERS_SUBJECT_OK            = "Il tuo rimborso è in arrivo!"
     NOTIFICATION_SERVICE_NOTIFY_PAYMENT_WINNERS_SUBJECT_KO            = "Si è verificato un problema con il tuo rimborso"
-    NOTIFICATION_SERVICE_NOTIFY_PAYMENT_WINNERS_SCHEDULER             = "-"
+    NOTIFICATION_SERVICE_NOTIFY_PAYMENT_WINNERS_SCHEDULER             = "-" #Send to Citizens a not of received tranfer files 0 */5 * * * ?
```

Comment:
Just typos but I think it's worth fixing it (I propose) this way:
# Send to Citizens a note of received transfer files

Reply:
@alberto-re-pagopa Thank you for pointing out, but I think the semantic is a bit different. The variable enables us to "notify citizens they received a wire transfer by Consap". "Files" is a typo